flexibleengine/acceptance/acceptance.go

```diff
@@ -21,6 +21,7 @@ var (
 	OS_ACCESS_KEY             = os.Getenv("OS_ACCESS_KEY")
 	OS_SECRET_KEY             = os.Getenv("OS_SECRET_KEY")
 	OS_PROJECT_ID             = os.Getenv("OS_PROJECT_ID")
+	OS_USER_ID              = os.Getenv("OS_USER_ID")
```

Comment:
use `ctrl + alt + L` to format the file

Reply:
![image](https://github.com/FlexibleEngineCloud/terraform-provider-flexibleengine/assets/145660317/e719fd61-9a31-4909-aae1-5299d30b1640)
done